src/components/EditProfileForm.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { Form, t } from '../components/Form';
+import Button from '../components/Button';
+import ProfileRequester from '../helpers/requesters/ProfileRequester'
+import BaseRequester from '../helpers/requesters/BaseRequester';
+
+class EditProfileForm {
+  constructor(props) {
+    super(props);
+
+    this._getInitialFormValues = this._getInitialFormValues.bind(this);
+    this._getFormType = this._getFormType.bind(this);
+    this._getFormOptions = this._getFormOptions.bind(this);
+    // this._clearFormErrors = this._clearFormErrors.bind(this);
+    this._onFormChange = this._onFormChange.bind(this);
+
+    this.state = {
+      formValues: this.getInitialFormValues(),
+      updating: true,
+      errors: [],
+    };
+  }
+
+  getInitialFormValues() {
+    return (
+      first_name: this.props.first_name,
+      last_name: this.props.last_name,
+      email: this.props.email,
+    );
+  }
+
+  getFormType() {
+    return t.struct({
+      firstName: t.String,
+      lastName: t.String,
+      email: t.String,
+    });
+  }
+
+  getFormOptions() {
```

Comment:
SInce you're using the underscored version `_getFormOptions` in the Form object and also binding that in the constructor this naming needs to be consistent all the way